Movie_MVP/Movie_MVP/Modules/DetailsMovieModule/Presenter/DetailsMoviePresenter.swift

```diff
@@ -0,0 +1,97 @@
+// DetailsMoviePresenter.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+protocol DetailsMovieViewProtocol: AnyObject {
+    func reloadData()
+}
+
+protocol DetailsMoviePresenterProtocol: AnyObject {
+    var movieID: Int? { get set }
+    func getCurrentFilm()
+    func getTitleCurrentFilm() -> String?
+    func getOverviewCurrentFilm() -> String?
+    func getPosterPathCurrentFilm() -> String?
+}
+
+final class DetailsMoviePresenter<DatabaseEntity: Database>: DetailsMoviePresenterProtocol {
```

Comment:
```suggestion
final class DetailsMoviePresenter: DetailsMoviePresenterProtocol {
```
некоему ему знать о <DatabaseEntity: Database>